force-app/main/default/objects/Template__c/fields/Role__c.field-meta.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Role__c</fullName>
```

Comment:
I don't believe this field is needed the template name would be enough I believe.